docs/source/user/aerodyn/theory_tailfin.rst

```diff
@@ -163,9 +161,24 @@ The user only needs to indicate the index `TFinAFIndex` within the list `AFNames
 Unsteady slender body model
 ---------------------------
 
-The unsteady slender body (USB) model is documented in :cite:`ad-hammam2022`.
+The unsteady aerodynamics of the tail fin is modeled based on Unsteady Slender Body Theory.
+The theory is extended to include the effect of high yaw angle :cite:`ad-hammam_NREL:2023`.
+
+The normal force on the tail fin can be described as
+
+.. math::  :label: TFUSBForce
+
+    N = \frac{\rho}{2} A_{tf} \bigg(  K_p x_1 V_{\text{rel},x} V_{\text{rel},y} +  \Big[x_2 K_v+(1- x_3)C_{Dc} \Big] V_{\text{rel},y}\big|V_{\text{rel},y}\big|\bigg)
+
+
+where :math:`\rho` is the density of air, :math:`A_{tf}` is the tail fin area, :math:`K_p` is the potential lift coefficient and :math:`K_v` is the vortex lift coefficient, and :math:`C_{Dc}` is the drag coefficient.
+:math:`x_i` are the separation functions calculated using a quasi-steady approximation as:
+
+.. math::  :label: TFUSBxiEquation
 
-The theory will be implemented and documented in a future release.
+    x_i = (1+exp{[\sigma_i (|\gamma_{tf}|-\alpha^*_i)]})^{-1}
 
 
+where :math:`\sigma_i` are emperical constants characterizing the decay of separation functions, :math:`\gamma_{tf}` is the yaw angle of the tail fin with respect to the free-stream wind (:math:`V_{\text{wind}}`), :math:`\alpha^*_i` are the characteristics angles for separation functions.
```

Comment:
We could add a small sentence here to mention what x1 takes value between 0 and 1,  and is used to activate or deactivate a given contributions to the normnal force (or something like that).